src/apiflask/scaffold.py

```diff
@@ -55,9 +55,14 @@ def handle_error(  # type: ignore
             error_headers,
         )
 
+    def load_location_data(self, schema: Schema, location: str) -> t.Any:
+        """
+        Expose the internal `_load_location_data` method to support loading data without validation
+        """
+        return self._load_location_data(schema=schema, req=flask_request, location=location)
```

Comment:
I was a little reluctant to rely on a non-public method from an external library, but since it's coming from within a child class, I think it might be acceptable, and I didn't see any other way to get access to the logic - I'm open to other suggestions here.